lib/conduit_web/templates/pipeline/show.html.eex

```diff
@@ -0,0 +1,12 @@
+<h2>Show Pipeline</h2>
+
+<ul>
+
+  <li>
+    <strong>Name:</strong>
+    <%= @pipeline.name %>
+  </li>
+
```

Comment:
Empty line? (also line 4)

Reply:
Will clean all these up with credo after I've got the Jenkins pipeline set up